src/ResourceManager/Network/Commands.Network.Test/packages.config

```diff
@@ -9,19 +9,22 @@
   <package id="Microsoft.Azure.Management.Authorization" version="2.0.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Compute" version="23.2.0" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.ContainerInstance" version="1.1.0" targetFramework="net452" />
-  <package id="Microsoft.Azure.Management.Network" version="19.7.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.KeyVault" version="2.4.3" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="19.6.2-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.OperationalInsights" version="0.19.0-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Redis" version="3.1.1-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.ResourceManager" version="1.9.1-preview" targetFramework="net452" />
   <package id="Microsoft.Azure.Management.Resources" version="2.20.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Management.Storage" version="6.1.0-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Test.Framework" version="3.0.0" targetFramework="net452" />
-  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.6.8" targetFramework="net45" />
+  <package id="Microsoft.Azure.Test.HttpRecorder" version="1.13.0" targetFramework="net452" />
```

Comment:
@akshaysngupta you should be able to revert this change, and even remove this reference to `HttpRecorder` since we pull it from NuGet elsewhere

Reply:
removed.